### ISSUE TITLE
Bug(react-devtools-inline@4.11.0): publish artifact points to /Users/bvaughn/Documents/git/react.devtools/

### ISSUE THREAD

**Issue by AviVahl** [facebook/react#21235]
Published `react-devtools-inline@4.11.0` has the following piece of code in `react-devtools-inline/dist/backend.js`:

```js
module.exports = require("/Users/bvaughn/Documents/git/react.devtools/packages/react-devtools-shared/src/config/DevToolsFeatureFlags.default");
```
:roll_eyes:

**Comment by AviVahl** [comment]
FYI @bvaughn

**Comment by bvaughn** [comment]
Huh. That's no good.

**Comment by bvaughn** [comment]
This is because I accidentally put the module map in an `externals` map rather than a `resolve.alias` map. Oops.

**Comment by bvaughn** [comment]
Fix published in 4.11.1

**Comment by AviVahl** [comment]
Thanks!
